packages/paypal-commerce-integration/src/paypal-commerce-credit/paypal-commerce-credit-payment-strategy.ts

```diff
@@ -215,4 +232,28 @@ export default class PayPalCommerceCreditPaymentStrategy implements PaymentStrat
             this.loadingIndicator.hide();
         }
     }
+
+    /**
+     *
+     * Render Pay Later Messages
+     *
+     * */
+    private renderMessages(paypalMessages: PayPalMessagesSdk, bannerContainerId: string): void {
+        if (bannerContainerId && document.getElementById(bannerContainerId)) {
```

Comment:
We don't need to check for `bannerContainerId` here, because it's always a string.

Reply:
`bannerContainerId` should be optional in this method, because `bannerContainerId` is not required in initialization options